mesonbuild/dependencies/pkgconfig.py

```diff
@@ -113,9 +132,11 @@ def list_all(self) -> ImmutableListProtocol[str]:
 class PkgConfigCLI(PkgConfigInterface):
     '''pkg-config CLI implementation'''
 
-    def __init__(self, env: Environment, for_machine: MachineChoice, silent: bool) -> None:
+    def __init__(
+        self, env: Environment, for_machine: MachineChoice, silent: bool, pkgbin: T.Optional[ExternalProgram] = None
+    ) -> None:
```

Comment:
Did this use the black deformatter? It looks deformed. :(

Do not indent the function arguments the same amount as the function *body*. Do not stick the closing parens for a function at the beginning of the line/block.

Please use:
```python
    def __init__(self, env: Environment, for_machine: MachineChoice, silent: bool,
                 pkgbin: T.Optional[ExternalProgram] = None) -> None:
```